test/engine/test_parseconnect.py

```diff
@@ -322,6 +322,14 @@ def test_query_dict_immutable(self, urlstr):
         with expect_raises_message(TypeError, ".*immutable"):
             url_obj.query["foo"] = "hoho"
 
+    def test_create_engine_url_invalid(self):
+        assert_raises_message(
```

Comment:
can this be a `with expect_raises_message:` please, thanks (see `test_query_dict_immutable` up above)

Reply:
Done